src/library-authoring/list-libraries/LibraryListPage.jsx

```diff
@@ -109,22 +109,22 @@ export class LibraryListPage extends React.Component {
 
     return (
       <>
-        <div className="wrapper-mast wrapper">
-          <header className="mast has-actions">
-            <ActionRow>
-              <h1 className="page-header">{intl.formatMessage(messages['library.list.page.heading'])}</h1>
-              <ActionRow.Spacer />
-              {libraries.count !== 0 && (
+        {libraries.count !== 0 && (
```

Comment:
This is changing behavior, so it's better if we have separate commit for it with a corresponding `feat:` or `fix:`.

Reply:
https://github.com/openedx/frontend-app-library-authoring/pull/83/commits/3d787cc610f1c24b63b30c978208189b022d00f9